### PR TITLE
[Feature, #5] - serialiser support and general cleanup

### PR DESCRIPTION
**Pull request by davesag**
The serialiser support has been added. I have also done a cleanup of the module names, file names etc based on some of the feedback in the issues and last PR.

This gem is now feature complete.

### CODE REVIEW

File: lib/datetime_helper/version.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module Labs
+  module DatetimeHelper
+    VERSION = "0.0.3"
```
Comment (gingermusketeer) [inline lib/datetime_helper/version.rb]: Should this be 1.0.0 for a feature complete gem?

Comment (davesag) [inline lib/datetime_helper/version.rb]: No.  It'll be 1.0.0 when its been code-reviewed, and any module / file naming issues have been cleaned up.

File: lib/datetime_helper.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
-require "westfield_datetime_helper/version"
+require "datetime_helper/version"
 
-module WestfieldLabs
+module Labs
```
Comment (gingermusketeer) [inline lib/datetime_helper.rb]: I think either `Labs` need to be removed or the gem name should be changed to `labs-datetime_helper` See http://guides.rubygems.org/name-your-gem/

Comment (davesag) [inline lib/datetime_helper.rb]: Okay.  Makes sense.  I'll ask @leondewey for a decision one way or the other.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
-A collection of useful utilities for projects that have to deal with dates, times, and timezones.
+A collection of useful utilities for projects that have to deal with dates, times, and timezones, with a focus on rails projects that enforce the use of Zulu Time.
 
-### Current Status: Rspec Matchers done
+### Current Status: Feature Complete
 
 1. The `rspec` matcher `be_zulu_time` has been implemeted.
 2. The `zulu_time` `ActiveModel` validator has been implemented.
+2. `in_zulu_time` ActiveModel::Serializer helper method has been implemented.
```
Comment (gingermusketeer) [inline README.md:10]: I would replace the current status section with 

```
### Features

1. `rspec` matcher `be_zulu_time` 
2. `zulu_time` `ActiveModel` validator 
3. `in_zulu_time` ActiveModel::Serializer helper
```

If i saw "Current Status: Feature Complete" I would read it as "Not open for PRs"

File: README.md [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
 
 #### Remaining to do
 
-2. `in_zulu_time` ActiveModel::Serializer support method
+1. Code review, and finalisation of the module names, class names, and file names
```
Comment (gingermusketeer) [inline README.md]: Is this going to be done as part of this PR or some time in the future

File: lib/datetime_helper.rb (hunk 2 above)
Comment (davesag) [inline lib/datetime_helper.rb]: Discussed with @leondewey - will remove the `Labs` wrapper.

File: README.md (hunk 3 above)
Comment (davesag) [inline README.md:10]: ok

File: README.md (hunk 4 above)
Comment (davesag) [inline README.md]: Next step but I'll pull it from the Readme now.

File: lib/datetime_helper.rb (hunk 2 above)
Comment (leondewey) [inline lib/datetime_helper.rb]: There is no need to brand the gem?

File: .travis.yml [hunk 5]
```diff
@@ -7,3 +7,6 @@ rvm:
   - 2.1.4
   - 2.2.0
   - 2.2.1
+before_install:
+- gem install bundler
+- export TZ=Australia/Canberra
```
Comment (gingermusketeer) [inline .travis.yml:12]: Is this so that it does not run in UTC?

Comment (davesag) [inline .travis.yml:12]: Yes. It's so the local_time tests are in somewhere other than UTC or else they fail.

**Comment by gwshaw** [comment]
LGTM

File: .travis.yml (hunk 5 above)
Comment (gingermusketeer) [inline .travis.yml:12]: Could add something like this to the rspec setup

```
around do |test|
  zone = ActiveSupport::TimeZone.all.sample
  Time.use_zone(zone, &test)
end
# OR
before do
  @original_zone = Time.zone
  zone = ActiveSupport::TimeZone.all.sample
  Time.zone = zone
end

after do
  Time.zone = @original_zone
end
```

Comment (davesag) [inline .travis.yml:12]: Nice, but adds a dependency on `ActiveSupport`.

**Pull request merged**